### ISSUE TITLE
What's your settings.cfg for generating the completion ground truth?

### ISSUE THREAD

**Issue by JennyVVV** [jbehley/voxelizer#3]
Hi, I want to increase the training data, but I'm not sure what your settings.cfg looks like, especially the ignored label ids.  Can you give me some suggestions?

Hope for your reply, and thank you in advance.

**Comment by jbehley** [comment]
If you want to extract scanwise, then the provided configuration, under `assets/semantic_kitti_dense.cfg` should do.
